src/app/app.component.html

```diff
@@ -7,6 +7,9 @@ <h1>Demo Application</h1>
     <option value="bmi">BMI Calculation</option>
     <option value="phq9">PHQ9 (no FHIRPath)</option>
   </select>
+  <button (click)="addTotalScoreRule()">
+    Export Scores
```

Comment:
"EXport Scopres" do not express well what i happening.  How about "Add Total Score Calculation"?

Reply:
Fixed